Makefile

```diff
@@ -0,0 +1,9 @@
+GHC := 9.2.4
```

Comment:
I did not follow the purpose of this file. 'Makefile' files are not used in the Stack repository.

Reply:
I was hoping for a lightweight way to maintain the cabal.config file. I'm open to other suggestions to do the same.